lib/ssh_scan/config_loader.rb

```diff
@@ -0,0 +1,31 @@
+require 'logger'
+
+require 'ssh_scan/config_loader/yaml'
+
+module SSHScan
+  module ConfigLoader
+    def self.load(config, class_name = nil, mode = 'file', logger = nil)
+      case mode
+        when 'file'
+          ext = File.extname(config)
+          case ext
+            when ".yaml", ".yml"
```

Comment:
Let's consider standardizing on one ext format for yaml files.  I don't care which one, but let's pick one.